src/SelfUpdateCommand.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
```

Comment:
Was the self:update command copied from Symfony? If so, please rewrite this copyright notice to make it clear that this is a derived work. Explicitly name the Symfony license, and identify the original copyright holder (Fabien Potencier) and the URL where the original code was obtained.

If this command merely used a Symfony command as boilerplate (e.g. from Symfony documentation), then this notice could probably just be removed.

Reply:
it is taken from magerun partially, but rewritten in the biggest parts, so I will rather remove that.

Reply:
Don't forget to update the header